src/GraphQL/Schemas/Primary/Substrate/Mutations/RetryTransactionsMutation.php

```diff
@@ -7,12 +7,14 @@
 use Enjin\Platform\GraphQL\Base\Mutation;
 use Enjin\Platform\GraphQL\Schemas\Primary\Substrate\Traits\InPrimarySubstrateSchema;
 use Enjin\Platform\Interfaces\PlatformGraphQlMutation;
-use Enjin\Platform\Models\Transaction;
+use Enjin\Platform\Models\Laravel\Transaction;
```

Comment:
Is there a reason to use this instead of the model resolver?

Reply:
When querying, it returns a Models\Laravel\Transaction, and it throws an exception here:
```
$txs->each(function (Transaction $tx): void {
```

I guess I can just omit the type

Reply:
Ah yes, in other places we either use `Model` or just omit.